CenterForOpenScience/osf.io
# Throttle user spam [ENG-1978]

## Purpose

Much of the spam that is created is created by one user in batches. If we can stop them early, it will slow them down.

## Changes

Checking for how much spam a user has created. Once they exceed the throttle, their account is disabled.
## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify that an account is disabled after making a lot of spam within 24 hours.
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised?

## Documentation

N/A

## Side Effects

Shouldn't be.
## Ticket


https://openscience.atlassian.net/browse/ENG-1978
